src/main/java/org/mdbenefits/app/cli/TransmissionCommands.java

```diff
@@ -171,12 +190,40 @@ private void transmitBatch(List<Transmission> transmissions) {
                 }
             }
 
-            // TODO - send emails to state (another ticket)
+            sendEmailToCaseworkers(transmission, confirmationNumber, emailRecipients, newFolder.getUrl(), errorMap);
 
             updateTransmissionStatus(transmission, TransmissionStatus.COMPLETED, errorMap, true);
         });
     }
 
+    // email addresses is a string of email addresses with a space in between
+    private void sendEmailToCaseworkers(Transmission transmission, String confirmationNumber, String emailAddresses,
```

Comment:
This has email address as the third parameter but above it's passed as newFolder.gerUrl()?

Reply:
Line 193 looks correct to me.  It's sending `emailRecipients` as the third parameter.